tasks/section_10_level1.yml

```diff
@@ -98,22 +98,18 @@
       - section10.4
 
   - name: 10.5.1 Lock Inactive User Accounts (check) (Scored)
-    command: grep INACTIVE /etc/login.defs
+    command: bash -c "useradd -D | grep -oP '(?<=^INACTIVE=)-?[0-9]+'"
```

Comment:
Good question. When I just did `useradd -D | grep -oP '(?<=^INACTIVE=)-?[0-9]+'`, for some reason it said that `-P` was not a valid option for `useradd`. I tried various methods of escaping the pipe and other things, but this is the only thing that worked.

I got the technique from http://stackoverflow.com/a/24685924. Seems like the `shell` module could also be a workaround.